spec/requests/api/v2/providers/contacts_spec.rb

```diff
@@ -1,6 +1,27 @@
 require "rails_helper"
 
-RSpec.describe "/api/v2/providers/<accredited_body_code>/contacts", type: :request do
```

Comment:
Should this file be moved to `spec/requests/api/v2/contacts_spec.rb`?

Also, on a related note, if we ever add an endpoint for listing contacts we'll probably want it to be scoped to the provider. But I guess that's a can we can kick down the road for now.

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1585/commits/594b4fef2b189ce4e88050843f22bfd93f6a13d6